R/gen_data.R

```diff
@@ -572,12 +613,41 @@ gen_data <- function(dictionary, varnames = "data_element_shortname", numcases =
       dis_output[dis_output$cluster_number == i, "household_id"] <- sample(1:(as.integer(nums/5) + 1), nums, replace = TRUE)
     }
 
-    # use household num as a standin for fact_0_id for now
-    dis_output$fact_0_id <- dis_output$household_id
+    ## create a var for eligible and interviewed
+
+    # drop interviewed and eligible
+    dis_output$eligible <- NULL
```

Comment:
given that you've copied this > 2 times means that we need to create a function

Reply:
i know but theres a lot that needs cleaning up in here as is. 
Lets fix everything first (including weighting) and then clean up gen_data

Reply:
I just really hate generating technical debt. I can fix this right now while it's still fresh in my mind.

Reply:
have at it ... just think we might have bigger fish to fry